include/triton/Target/LLVMIR/LLVMIRTranslation.h

```diff
@@ -29,6 +29,8 @@ translateTritonGPUToLLVMIR(llvm::LLVMContext *llvmContext,
 std::unique_ptr<llvm::Module>
 translateLLVMToLLVMIR(llvm::LLVMContext *llvmContext, mlir::ModuleOp module);
 
+bool linkLibdevice(llvm::Module &module, const std::string &path);
```

Comment:
StringRef seems better here, for it avoid creation of a std::string instance when passing a `char[]` as path.

Reply:
Good point, let me change.